edi_voxel_secondary_unit_oca/views/product_secondary_unit_views.xml

```diff
@@ -2,15 +2,17 @@
 <!-- Copyright 2021 Tecnativa - Ernesto Tejeda
      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
 <odoo>
-    <record id="product_secondary_unit_view_tree" model="ir.ui.view">
-        <field name="name">product.secondary.unit.view.tree.inherit</field>
-        <field name="model">product.secondary.unit</field>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
         <field
             name="inherit_id"
-            ref="product_secondary_unit.product_secondary_unit_view_tree"
+            ref="product_secondary_unit.product_template_form_view"
```

Comment:
I assume that some changes have been made in the base module that cause the change from one view to another